content/en/docs/installing-on-linux/introduction/air-gapped-installation.md

```diff
@@ -1,670 +1,555 @@
 
-{{</ notice >}} 
+   - **kubesphere.tar.gz**: Specifies the image package of the source cluster.
 
-## Step 6: Create a Cluster
+    {{</ notice >}}
 
-In this tutorial, KubeSphere is installed on multiple nodes, so you need to specify a configuration file to add host information. Besides, for air-gapped installation, pay special attention to `.spec.registry.privateRegistry`, which must be set to **your own registry address**. See the [complete YAML file](../air-gapped-installation/#2-edit-the-configuration-file) below for more information.
-
-### Create an example configuration file
-
-Execute the following command to generate an example configuration file for installation:
-
-```bash
-./kk create config [--with-kubernetes version] [--with-kubesphere version] [(-f | --file) path]
-```
-
-For example:
+5. Create a Harbor project.
+   
+   {{< notice note >}}
 
-```bash
-./kk create config --with-kubernetes v1.21.5 --with-kubesphere v3.2.1 -f config-sample.yaml
-```
+   As Harbor adopts the Role-based Access Control (RBAC) mechanism, which means that only specified users can perform certain operations. Therefore, you must create a project before pushing images to Harbor. Harbor supports two types of projects:
+
+   - **Public**: All users can pull images from the project.
+
+   - **Private**: Only project members can pull images from the project.
+
+   The username and password for logging in to Harbor is **admin** and **Harbor12345** by default. The installation file of Harbor is located in **/opt/harbor**, where you can perform O&M of Harbor.
+
+
+  
+    {{</ notice >}}
+
+   Method 1: Run the following commands to create a Harbor project.
+
+   a. Run the following command to download the specified script to initialize the Harbor registry:
+
+      ```bash
+      curl https://github.com/kubesphere/ks-installer/blob/master/scripts/create_project_harbor.sh
+      ```
+
+   b. Run the following command to modify the script configuration file:
+
+      ```bash
+      vim create_project_harbor.sh
+      ```
+
+      ```yaml
+      #!/usr/bin/env bash
+      
+      # Copyright 2018 The KubeSphere Authors.
+      #
+      # Licensed under the Apache License, Version 2.0 (the "License");
+      # you may not use this file except in compliance with the License.
+      # You may obtain a copy of the License at
+      #
+      #     http://www.apache.org/licenses/LICENSE-2.0
+      #
+      # Unless required by applicable law or agreed to in writing, software
+      # distributed under the License is distributed on an "AS IS" BASIS,
+      # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+      # See the License for the specific language governing permissions and
+      # limitations under the License.
+      
+      url="https://dockerhub.kubekey.local"  #Change the value of url to https://dockerhub.kubekey.local.
+      user="admin"
+      passwd="Harbor12345"
+      
+      harbor_projects=(library
```

Comment:
Please refer to: https://github.com/kubesphere/ks-installer/blob/master/scripts/create_project_harbor.sh